src/Jupyter/Magic/Simulate.cs

```diff
@@ -33,15 +36,9 @@ public async Task<ExecutionResult> RunAsync(string input, IChannel channel)
             var symbol = SymbolResolver.Resolve(name) as IQSharpSymbol;
             if (symbol == null) throw new InvalidOperationException($"Invalid operation name: {name}");
 
-            using (var qsim = new QuantumSimulator())
-            {
-                qsim.DisableLogToConsole();
-                qsim.OnLog += channel.Stdout;
-
-                var value = await symbol.Operation.RunAsync(qsim, args);
-
-                return value.ToExecutionResult();
-            }
+            using var qsim = new QuantumSimulator().WithJupyterDisplay(channel, ConfigurationSource);
```

Comment:
is this new syntax for `using`? you're way more up to date than me.

Reply:
The semantics of `using var` are identical to the old `using`, but implicitly places the closing `}` for the `using` statement at the end of the current scope. I find it pretty useful for when I want to have a `using` for most or all of a method scope.